build/SharedFx.targets

```diff
@@ -383,4 +422,45 @@
   </Target>
 
   <Target Name="BuildSharedFx" DependsOnTargets="ResolveSharedFxFiles;CrossGenAssemblies;CrossGenSymbols;PackSharedFx"/>
+
+  <Target Name="_BuildSharedFxSymbols">
+    <!-- Clear working directory -->
+    <RemoveDir Directories="$(_WorkRoot)" />
+
+    <ItemGroup>
+      <RuntimeSymbolsArchive Include="win-x86;win-x64;osx-x64;linux-x64"/>
+    </ItemGroup>
+
+    <!-- Extract Symbols archives -->
+    <UnzipArchive
+      File="$(_SymbolsSourceDir)$(SymbolsSourceArchiveName)-%(RuntimeSymbolsArchive.Identity).zip"
```

Comment:
Can you clarify what's going on here? Why are we producing a zip of symbols, unzipping it, and then packing it into a nuspec?

Reply:
The symbols are built on each platform win-x64/win-x86/linux-x64/osx-x64. Then we need another configuration that takes the symbols produced on each platform and pack them into a symbols nupkg for .App and .All.

Reply:
Maybe we should rid split our symbols packages... I'm thinking ahead to our goal to build aspnetcore on a single machine. If we can avoid introducing more build outputs that require building on 3 machines and still satisfy the symbols package requirement, I think we should do that.